app/src/main/java/org/schabi/newpipe/local/bookmark/BookmarkFragment.java

```diff
@@ -256,6 +259,60 @@ private void showRemoteDeleteDialog(final PlaylistRemoteEntity item) {
     }
 
     private void showLocalDialog(final PlaylistMetadataEntry selectedItem) {
+        final String rename = getString(R.string.rename);
+        final String delete = getString(R.string.delete);
+        final String unsetThumbnail = getString(R.string.unset_playlist_thumbnail);
+        final boolean isThumbnailPermanent = localPlaylistManager
+                .getIsPlaylistThumbnailPermanent(selectedItem.uid);
+
+        final AlertDialog.Builder builder = new AlertDialog.Builder(activity);
+
+        final ArrayAdapter<String> arrayAdapter = getLocalDialogArrayAdapter(isThumbnailPermanent,
+                unsetThumbnail);
+        arrayAdapter.addAll(rename, delete, unsetThumbnail);
```

Comment:
Mmmh, after trying out the app I think it's better if you just don't add the "unsetThumbnail" entry instead of disabling it. At the moment the disabled item is still clickable and once clicked closes the dialog, which feels strange. Also, it's expected across the app for things to appear only when they do make sense.

Reply:
I'd prefer to go with simplicity of implementation here, so that there is less code that may fail. As I said there is no big difference anyway from the user perspective (grayed out things are rarely useful to find out features, as they don't point to anything, imho). So go with the simple approach of just not adding the `unsetThumbnail` when it does not apply.